### PR TITLE
chore: added resource for tailwindCSS in UI generators

### PR DESCRIPTION
**Pull request by kirtiraj22**
## Changes proposed

- Added [Tailwind Awesome](https://www.tailwindawesome.com/?technology=3) to Frontend -> UI Generators

## Screenshots
Screenshot (171)

## Note to reviewers
This PR is a part of **GSSoC'23**

### CODE REVIEW

File: database/frontend/ui-generators.json [hunk 1]
```diff
@@ -159,5 +159,12 @@
     "url": "https://ui.glass/generator/",
     "category": "frontend",
     "subcategory": "ui-generators"
+  },
+  {
+    "name":"Tailwind Awesome",
+    "description":"Tailwind Awesome is a curated list of the best Tailwind templates & UI kits in the internet.",
```
Comment (CBID2) [inline database/frontend/ui-generators.json:165]: [Typo]
```suggestion
    "description":"Tailwind Awesome is a curated list of the best Tailwind templates & UI kits on the internet.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Found a typo that needs to be changed.

**Comment by kirtiraj22** [comment]
Hey @CBID2, I have written the description based on the information provided on the Tailwind Awesome website itself!

you can checkout the website's description here [Tailwind Awesome](https://www.tailwindawesome.com/?technology=3)

**Comment by kirtiraj22** [comment]
hey @CBID2, I have made the required changes kindly review it!

**Comment by CBID2** [comment]
> Hey @CBID2, I have written the description based on the information provided on the Tailwind Awesome website itself!
> 
> 
> 
> you can check out the website's description here [Tailwind Awesome](https://www.tailwindawesome.com/?technology=3)

Hey @kirtiraj22. I saw the link you provided. I still recommend making the change. More often than not, a website’s content doesn't always present things correctly whether it’s grammar or information about a topic, so it's crucial to critically evaluate it before accepting it. 😄

**Review by CBID2 — APPROVED** [review]
LGTM! Remember, question the information you see as you find more resources! :)

**Comment by kirtiraj22** [comment]
> Collaborator
Apologies, I will ensure to be more cautious and evaluate the content thoroughly next time. 😄

**Review by rupali-codes — APPROVED** [review]
thanks, lgtm

**Pull request merged**